packages/manager/apps/pci-load-balancer/src/api/hook/useAddons.ts

```diff
@@ -1,32 +1,41 @@
-import { useCatalog } from '@ovh-ux/manager-pci-common';
+import { TAddon, useCatalog } from '@ovh-ux/manager-pci-common';
 import { useMemo } from 'react';
-import { TAddon } from '@/pages/create/store';
 
-const SIZE_FLAVOUR_REGEX = /octavia-loadbalancer.loadbalancer-([23]?xl|[sml]).hour.consumption/;
+const SIZE_FLAVOUR_REGEX = /octavia-loadbalancer.loadbalancer-([23]?xl|[sml]).hour.consumption$/;
+
+const getSize = (addon: TAddon) => addon.product.split('-').pop();
```

Comment:
I would prefer getting the last element instead of the pop method, you can use slice(-1) for example

The pop method modifies the original list so it causes memory changes